protos/telemetry/telemetry.proto

```diff
@@ -640,6 +658,26 @@ message RcStatus {
     float signal_strength_percent = 3 [(mavsdk.options.default_value)="NaN"]; // Signal strength (range: 0 to 100, NaN if unknown)
 }
 
+// Cellular modem status type.
+message CellularStatus {
+    uint32 id = 1;// (actually uint8_t)
```

Comment:
I'd say there is no need to specify "actually uint8_t". The MAVSDK API does not have to explain the MAVLink implementation details. However, it would be nice to describe what those fields are, because this will be part of the generated documentation :-)